src/pages/FanPage/FanPage.tsx

```diff
@@ -198,7 +223,7 @@ export class FanPage extends React.Component<IFanPageProps, IFanPageState> {
 
 
 
-      <WhenShouldDaveRestSection />
+      <WhenShouldDaveRestSection mo2={this.state.mo2} coreBodyTemp={this.state.coreBodyTemp}/>
```

Comment:
more than one prop should cause the component to go on multiple lines for readability:
```
<WhenShouldDaveRestSection 
    mo2={this.state.mo2} 
    coreBodyTemp={this.state.coreBodyTemp} />
```

Reply:
Done!